build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -181,8 +185,8 @@ CONFIG_AMEBASMART_TRUSTZONE=y
 # CONFIG_ARCH_NOINTC is not set
 # CONFIG_ARCH_VECNOTIRQ is not set
 # CONFIG_ARCH_DMA is not set
-# CONFIG_ARCH_HAVE_IRQPRIO is not set
-CONFIG_ARCH_L2CACHE=y
+CONFIG_ARCH_HAVE_IRQPRIO=y
```

Comment:
You don't enable this in the flat. I think this should be the same regardless of flat or loadable.

Reply:
Enabled in all defconfigs, added default selection for Cortex A32 as well.